homework_6/package_2/Car.java

```diff
@@ -0,0 +1,19 @@
+package package_2;
+
+public class Car {
+    private void startElectricity(){
+        System.out.println("Start electricity");
+    }
+    private void startCommand(){
+        System.out.println("Start command");
+    }
+    private void startFuelSystem(){
+        System.out.println("Start fuel system");
+    }
+    public void start(){
```

Comment:
Public methods should be declared first and private at the end of the class. Please reorder methods accordingly.